pkgs/development/compilers/ante/default.nix

```
@@ -0,0 +1,47 @@
+{ fetchFromGitHub
+, lib
+, libffi
+, libxml2
+, llvmPackages_13
+, ncurses
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage {
+  pname = "ante";
+  version = "unstable-2022-08-22";
+  src = fetchFromGitHub {
+    owner = "jfecher";
+    repo = "ante";
+    rev = "8b708d549c213c34e4ca62d31cf0dd25bfa7b548";
+    sha256 = "sha256-s8nDuG32lI4pBLsOzgfyUGpc7/r0j4EhzH54ErBK7A0=";
+  };
+  cargoSha256 = "sha256-29D7kPG7vop9lIxWQnaHkCTRY8YsCjERRCOvbU7oemQ=";
+
+  nativeBuildInputs = [ llvmPackages_13.llvm ];
+  buildInputs = [ libffi libxml2 ncurses ];
+
+  postPatch = ''
+    substituteInPlace tests/golden_tests.rs --replace \
+      'target/debug' "target/$(rustc -vV | sed -n 's|host: ||p')/release"
+  '';
+  preBuild =
+    let
+      major = lib.versions.major llvmPackages_13.llvm.version;
+      minor = lib.versions.minor llvmPackages_13.llvm.version;
+      llvm-sys-ver = "${major}${builtins.substring 0 1 minor}";
+    in
+    ''
+      export LLVM_SYS_${llvm-sys-ver}_PREFIX=${llvmPackages_13.llvm.dev}
+      export ANTE_STDLIB_DIR=$out/lib;
```

```suggestion
      export ANTE_STDLIB_DIR=$out/lib
```
This is not required, also for the other lines.

---

Yeah, should commit the changes form the code review here, or just rebase and force push?